ArduCopter/events.cpp

```diff
@@ -293,10 +293,15 @@ void Copter::gpsglitch_check()
         ap.gps_glitching = gps_glitching;
         if (gps_glitching) {
             AP::logger().Write_Error(LogErrorSubsystem::GPS, LogErrorCode::GPS_GLITCH);
-            gcs().send_text(MAV_SEVERITY_CRITICAL,"GPS Glitch");
+            // GPS glitch can also be caused by compass glitch
+            if(filt_status.flags.compass_glitch){
```

Comment:
nitpick: can you add a space between the "if" and the bracket and then again between the bracket and the, "{"?

Reply:
nitpick: can you also change the commit title to start with "Copter:" instead of "ArduCopter:"?  Sorry to be nitpicky..